controller/users.go

```diff
@@ -418,7 +417,7 @@ func LoadKeyCloakIdentities(appl application.Application, users []account.User)
 		identity, err := loadKeyCloakIdentity(appl, user)
 		// if we can't find the Keycloak identity
 		if err != nil {
-			logrus.Error(err)
+			log.Error(nil, map[string]interface{}{"user": user, "err": err}, "unable to load user keycloak identity")
```

Comment:
I believe this is done on purpose cause we are trying to get the identity of a list of users. If we don't have it the user won't be added. But we can always ask to `git blame` :)